pyblazing/pyblazing/apiv2/context.py

```diff
@@ -793,6 +794,9 @@ def __init__(self, dask_client=None, network_interface=None, allocator="managed"
                                     BLAZING_DEVICE_MEM_RESOURCE_CONSUMPTION_THRESHOLD : The percent (as a decimal) of total GPU memory that the memory resource
                                             will consider to be full
                                             default: 0.95
+                                    BLAZING_HOST_MEM_RESOURCE_CONSUMPTION_THRESHOLD : The percent (as a decimal) of total host memory that the memory resource
```

Comment:
We have the ability to set config_options during BlazingContext creation and also when running .sql(). We should mention in these comments that BLAZING_HOST_MEM_RESOURCE_CONSUMPTION_THRESHOLD and BLAZING_DEVICE_MEM_RESOURCE_CONSUMPTION_THRESHOLD can only be set in BlazingContext creation

Reply:
Added endnote.